### ISSUE TITLE
Update of the https://popmotion.io/learn/morph-svg/  higly needed

### ISSUE THREAD

**Issue by d4g0** [Popmotion/popmotion#899]
**Problem**
Since the tween api is [deprecated](https://github.com/Popmotion/popmotion/issues/898) 

**Solution**
I think will be nice update the docs where appears like https://popmotion.io/learn/morph-svg/ 
Can i leverage the tween job with animate for this particular example ?

**Comment by dayvista** [comment]
I am curious about what replaces tween as well. I tried animate but got a type error when compiling.

**Comment by aellerton** [comment]
I’m attempting to upgrade from popmotion 8.6.* to the latest and ran into the issue of "The Missing Tween".

That is, the tween function isn't found but docs and example code still refer to it.

Can someone point me in the right direction?

**Comment by maximilliangeorge** [comment]
Yep, when searching from Google I find the old docs (v8). The new docs (v9) can be found here: https://popmotion.io/

Version 9 is a slimmed down version with a different API entirely. For what it's worth, rebranding v9 with a new name might be a good idea? I'm in love with popmotion v8, which currently has more features and flexibility. I'm sure there's a good reason for creating a new API, but it can really confuse both new and old users with so many breaking changes.
